.github/workflows/basic.yml

```diff
@@ -226,7 +226,6 @@ jobs:
         -B ${{env.BUILD_DIR}}
         ${{matrix.toolset}}
         -DCMAKE_PREFIX_PATH="${{env.VCPKG_PATH}}"
-        -DCMAKE_BUILD_TYPE=${{matrix.build_type}}
```

Comment:
I guess there might be more of such entries (e.g. in benchmarks)

Reply:
I've verified that it's the only place where this variable is used for Windows tests in the CI.

Reply:
I think, it is still in benchmarks:

https://github.com/oneapi-src/unified-memory-framework/actions/runs/8421036255/job/23057332971#step:6:54

Reply:
Oh, thanks!